src/components/organizations/new/index.js

```diff
@@ -0,0 +1,51 @@
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+
+import {CreateOrganizationForm} from './form'
+import {LocalAddMembers} from './members'
+import Container from 'gComponents/utility/container/Container.js'
+
+import './style.css'
+
+function mapStateToProps(state) {
+  return {
+    newOrganization: state.newOrganization,
+  }
+}
+
+@connect(mapStateToProps)
+export class CreateOrganizationPageContainer extends Component {
+  componentWillMount() {
+    this.props.dispatch({type: 'CLEAR_NEW_ORGANIZATION'})
+  }
+  componentWillUnmount() {
+    this.props.dispatch({type: 'CLEAR_NEW_ORGANIZATION'})
+  }
+  render() { return (<CreateOrganizationPage newOrganization={this.props.newOrganization}/>) }
+}
+
+export const CreateOrganizationPage = ({newOrganization}) => {
```

Comment:
No reason to have a separate functional component for this. It is literally directly inserted into your render above.